packages/opentelemetry-cloud-trace-propagator/package.json

```diff
@@ -22,5 +31,22 @@
     "doc",
     "LICENSE",
     "README.md"
-  ]
+  ],
+  "devDependencies": {
+    "@opentelemetry/context-base": "0.9.0",
+    "@types/mocha": "7.0.2",
+    "@types/node": "12.12.47",
+    "codecov": "3.7.0",
+    "gts": "1.0.0",
+    "mocha": "7.2.0",
+    "nyc": "15.1.0",
+    "rimraf": "3.0.2",
+    "ts-mocha": "7.0.0",
+    "typescript": "3.9.3"
+  },
+  "dependencies": {
+    "@opentelemetry/api": "^0.9.0",
+    "@opentelemetry/core": "^0.9.0",
+    "hex2dec": "^1.0.1"
```

Comment:
Is node 8 going to be supported much longer by `opentelemetry-operations-js`?

If not you could consider getting rid of this dependency and use [`BigInt`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt) instead, which is supported since node 10.4.

Reply:
I am not sure about this. If you don't mind I would like to handle this in separate PR.

Reply:
We should probably stay in sync with the OpenTelemetry SDK which still supports Node 8: https://github.com/open-telemetry/opentelemetry-js#browser-support